package.json

```diff
@@ -8,13 +8,21 @@
   "scripts": {
     "dev": "parcel *.html",
     "build": "parcel build *.html --public-url './'",
-    "deploy": "yarn build && gh-pages -d dist"
+    "deploy": "yarn build && gh-pages -d dist",
+    "format": "prettier --write js/*.js",
+    "lint": "eslint js/*.js",
+    "postinstall": "cp githooks/* .git/hooks || true"
```

Comment:
На Маке не срабатывает хук, потому что он не исполняемый:
> hint: The '.git/hooks/pre-commit' hook was ignored because it's not set as executable.
> hint: You can disable this warning with `git config advice.ignoredHook false`.

См. https://gist.github.com/ipetropolsky/de00874119d6622be969b6af265da3aa#автоформатирование-при-коммите

Reply:
Добавил в postinstall параметр `-p`, чтобы права доступа сохранялись и `chmod +x githooks/pre-commit`

Reply:
Что-то у меня до сих пор не исполняемый.